nab/detectors/skyline/algorithms.py

```diff
@@ -198,6 +199,8 @@ def histogram_bins(timeseries):
   return False
 
 
+# The method below is excluded because it uses floating point time whereas NAB
+# uses datetime timestamp
```

Comment:
Yes there was another reason as well. I modified ks_test a little and made it work with NAB timestamp but it was slow like grubbs, so ended up terminating the run before completion. Should I change the reason to 'computationally inefficient' instead?